frontend/rust-lib/flowy-database2/src/services/group/controller_impls/select_option_controller/util.rs

```diff
@@ -171,6 +174,15 @@ pub fn make_inserted_cell(group_id: &str, field: &Field) -> Option<Cell> {
       let cell = insert_url_cell(group_id.to_owned(), field);
       Some(cell)
     },
+    FieldType::DateTime => {
+      let date = NaiveDateTime::parse_from_str(
+        &format!("{} 00:00:00", group_id).to_string(),
+        "%Y/%m/%d %H:%M:%S",
```

Comment:
I think this hard code parser might be a problem. How about reading the date format from the TypeOption? The title of the group might be the format string using the TypeOption format.

Reply:
Correct. I will fix it.

Reply:
Looking at this again, It doesn't seem to be a problem because I'm using the timestamp for `insert_date_cell`.

Reply:
@appflowy just wanted to mention if this isn't a problem anymore I think this PR can get merged.